aries/aries_vcx_core/src/wallet2/indy_wallet/mod.rs

```diff
@@ -0,0 +1,52 @@
+use super::BaseWallet2;
+use crate::wallet::indy::IndySdkWallet;
+
+pub mod indy_did_wallet;
+pub mod indy_record_wallet;
+
+const WALLET_OPTIONS: &str =
+    r#"{"retrieveType": true, "retrieveValue": true, "retrieveTags": true}"#;
+
+const SEARCH_OPTIONS: &str = r#"{"retrieveType": true, "retrieveValue": true, "retrieveTags": true, "retrieveRecords": true}"#;
+
+impl BaseWallet2 for IndySdkWallet {}
+
+#[allow(dead_code)]
+pub(crate) mod test_helper {
```

Comment:
You might want to place this into `test_utils` crate

Reply:
Though I am noticing this is not used in this PR - is this actually a dead code, or used by somewhere in the subsequent PRs?

It seems like this is some util code specific for the one and only mysql test suite `test_mysql_wallet`

Reply:
It is used in all tests to set up wallet, though compiler sees it as unused for some reason.